dts/bindings/input/st,stmpe811.yaml

```diff
@@ -0,0 +1,133 @@
+    type: int
+    default: 0
+    description: |
+      Signed raw X axis start for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: no offset.
+
+  raw-y-min:
+    type: int
+    default: 0
+    description: |
+      Signed raw Y axis start for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: no offset.
+
+  raw-x-max:
+    type: int
+    default: 4096
+    description: |
+      Raw X axis end for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: raw touchscreen resolution.
+
+  raw-y-max:
+    type: int
+    default: 4096
+    description: |
+      Raw Y axis end for scaling the reported coordinates.
+      No effect if screen size is not set.
+      Default: raw touchscreen resolution.
+
+  panel-driver-settling-time-us:
+    type: int
+    enum:
+      - 10
+      - 100
+      - 500
+      - 1000
+      - 5000
+      - 10000
+      - 50000
+      - 100000
+    required: true
+    description: |
+      Panel driver settling time (microseconds). For large panels (> 6"), a capacitor of 10 nF
+      is recommended at the touchscreen terminals for noise filtering.
+      As a general rule, 1-5 nF capacitors require around 500 us settling time, and 5-10 nF need
+      around 1 ms. When a larger capacitor is used, this value should be changed, as it can
+      lead to inaccuracy of the measurement.
+
+  touch-detect-delay-us:
+    type: int
+    enum:
+      - 10
+      - 50
+      - 100
+      - 500
+      - 1000
+      - 5000
+      - 10000
+      - 50000
+    default: 5000
+    description: |
+      Touch detect delay (microseconds) is the delay from the activation of the pull-up resistor
+      in the X+ line to the time the device performs touch detection.
+      The default value provides enough time for filtering capacitors up to about 50 nF,
+      and above this value, it should be increased.
+      If no capacitor, or a smaller capacitor is used, this value can be lowered to
+      minimize detection latency, but it could lower the position stability.
+
+  touch-average-control:
+    type: int
+    enum:
+      - 1
+      - 2
+      - 4
+      - 8
+    default: 8
+    description: |
+      Average control (number of samples).
+      The default value helps to reduce the effect of surrounding noise
+      and can be lowered to minimize detection latency at the cost of
+      less filtering.
+
+  tracking-index:
+    type: int
+    enum:
+      - 0
+      - 4
+      - 8
+      - 16
+      - 32
+      - 64
+      - 92
+      - 127
+    default: 127
+    description: |
+      Tracking index determines the minimal distance between
+      the current touch position and the previous touch position.
+      If the distance is shorter than the tracking index, it is discarded.
+      The default value reduces the number of interrupts to handle
+      and can be lowered if more frequent readings are needed.
```

Comment:
-1

Reply:
Hey, I don't know what `-1` means, I just assume you have some problem with the 2 lines. You can specify what you require, it will be better for both of us. 
We have moved those properties to dts, but that is just an assumption which I would prefer to avoid.